templates/none/routes/index.js

```diff
@@ -0,0 +1,10 @@
+var express = require('express');
+var router = express.Router();
+
+router.get('/', function (req, res, next) {
+  res.sendFile('index.html', {
```

Comment:
I've left it for now. This change would make the `none` view engine more different than the rest of the view engines. I'm up for changing it if `express.static` is more desirable or more idiomatic Express.